test/schema_generators/generators_test.cljc

```diff
@@ -1,19 +1,20 @@
 (ns schema-generators.generators-test
   #?(:clj (:use clojure.test))
   (:require
-   #?(:cljs [cljs.test :refer-macros [deftest is testing run-tests]])
-   #?(:cljs [cljs.reader :refer [read-string]])
-   [clojure.test.check]
-   [clojure.test.check.properties :as properties :include-macros true]
-   [clojure.test.check.generators :as check-generators]
-   [clojure.test.check.clojure-test #?@(:clj [:refer [defspec]]
-                                        :cljs [:refer-macros [defspec]])]
-   [schema.core :as s :include-macros true]
-   [schema-generators.generators :as generators]))
+    #?(:cljs [cljs.test :refer-macros [deftest is testing run-tests]])
```

Comment:
Can we remove the spurious whitespace changes please?

Reply:
Changes on https://github.com/plumatic/schema-generators/pull/29/commits/7043572018c08c519a1fa712d9f3ef7834acb3b6 and https://github.com/plumatic/schema-generators/pull/29/commits/3bc12dcfb35437788737f4b5dbf311d805fd4e2f